db/migrate/20200311135425_clear_approved_users_from_the_review_queue.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class ClearApprovedUsersFromTheReviewQueue < ActiveRecord::Migration[6.0]
+  def up
+    stuck_user_ids = User.joins("INNER JOIN reviewables r ON users.id = r.target_id")
```

Comment:
Can we replace this with SQL queries? We've found that using AR objects often introduces failures in the future when the structure of the objects change (or they are removed!)

Reply:
Done!